packages/flagship/src/add-keys-android.ts

```diff
@@ -19,7 +19,7 @@ const buildConfig =
   {};
 
 const gradlePropertiesPath =
-  getOption('gradlePropertiesPath') || `../../../../android/gradle.properties`;
+  getOption('gradlePropertiesPath') || `../android/gradle.properties`;
```

Comment:
This is intended to modify the "ejected" android folder that gets copied to the project root when flagship init is run. Because add-keys-android is typically run after init this will be modifying the android folder inside of flagship instead of the android folder in the root project directory.